configure.ac

```diff
@@ -548,6 +548,14 @@ AM_COND_IF(BUILDOPT_SYSTEMD_AND_LIBMOUNT,
   AC_DEFINE([BUILDOPT_LIBSYSTEMD_AND_LIBMOUNT], 1, [Define if systemd and libmount]))
 if test x$with_libsystemd = xyes; then OSTREE_FEATURES="$OSTREE_FEATURES systemd"; fi
 
+AC_ARG_WITH(modern-grub,
```

Comment:
Hmm, I wonder if this should just be e.g. `--with-grub-2.02` instead. (Not sure Autoconf supports periods in these switches.) Today's modern GRUB is tomorrow's ancient GRUB. :)

Or maybe we should be specific about the feature this is enabling, which might be safer given that each distro carries so many patches. E.g. `--without-grub-efi-16-suffixes`?

Reply:
There's also the argument to flip the default now - rip off the band-aid and require people who need it (Endless and probably Fedora derivatives for now to be safe) to explicitly pass `--with-grub-efi-16` or.

Reply:
Though I'm conflicted on this myself...if we all agree the real end goal is everyone has a grub that does BLS, then maybe it's not worth the disruption from this.

> Today's modern GRUB is tomorrow's ancient GRUB. :)

True in general but the idea here is we won't care about the grub version in the future.

So...I know this is all very bike-sheddable but I am circling in on just accepting `--with-modern-grub` as is.